server-src/src/main/java/com/itt/kmt/services/UserService.java

```diff
@@ -128,15 +128,11 @@ public User updateUser(final User user, final String id) {
             throw new RuntimeException("user is not active");
         } else if (existingUser != null) {
 
-            if (!user.getFirstName().isEmpty()) {
-                existingUser.setFirstName(user.getFirstName());
-            }
-            if (!user.getLastName().isEmpty()) {
-                existingUser.setLastName(user.getLastName());
-            }
-            if (!user.getUserRole().isEmpty()) {
-                existingUser.setUserRole(user.getUserRole());
-            }
+            existingUser.setFirstName(user.getFirstName());
```

Comment:
Please revert these changes
We want to check validity of data before writing data to DB

Reply:
We already added the validator for user add/update that will take care of validations of it, no need to recheck here